_conferences/devfest-singapore-2019.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "DevFest Singapore 2019"
```

Comment:
This should just be "GDG DevFest" to match the others. The location and date are already displayed adjacent to the name.

Reply:
Done.

Reply:
Thanks!